website/videoBlog/2023-08-03-Powering_EventDriven_Workloads_with_Hudi_Read_Stream_AWS_Glue_Streaming_JOBS.md

```diff
@@ -14,6 +14,6 @@ tags:
 - streaming
 - near real-time analytics
 - event bus
-- amazon sqs
+- aws sqs
```

Comment:
Lets leave it as amazon instead of aws. The reason, I added this tag is based on how the sqs documentation calls it. For ex: https://aws.amazon.com/sqs/ names it `Amazon SQS` not `aws sqs`.